### PR TITLE
Update stream

### PR DESCRIPTION
**Pull request by TSterling76**

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -11,3 +11,6 @@ summary: '"This Security Technical Implementation Guide is published as a tool
   to this document should be sent via email to the following address:
   disa.stig_spt@mail.mil."'
 version: 2.6.0
+
+# administrator-approved extension IDs
```
Comment (aaronlippold) [inline inspec.yml]: Did you want to make these inputs?

aka

```yaml
input:
  - name: my_input
    type: string
    required: false
    description: "my human desc of what this is for"
    value: "my defualt value"
 ```
documented here: https://docs.chef.io/inspec/inputs/

**Review by aaronlippold — CHANGES_REQUESTED** [review]
Good start just one question on the `inspec.yml`

File: inspec.yml (hunk 1 above)
Comment (TSterling76) [reply inspec.yml]: I can make them, I just wanted to know if I was on the right path.

**Comment by TSterling76** [comment]
Just added some changes,

**Pull request merged**